geoengine/workflow.py

```diff
@@ -464,12 +465,18 @@ def save_as_dataset(
 
         return Task(TaskId.from_response(response))
 
+    # pylint: disable=too-many-locals
```

Comment:
> Having too many locals may indicate that you're doing too much in a function and that classes regrouping some attributes could be created. Maybe operations could be separated in multiple functions. Are all your variables really closely related ?

I guess it's time to react on that.

Reply:
but are there really too many variables? I think it's a false positive due to the nested functions inside the function.

Reply:
maybe we should unnest those functions, then

Reply:
but they capture stuff from the caller. If I unnest them they need more params and maybe also different logic. I'd rather not.

Reply:
I don't see that they capture stuff.

The thing is, if you just disable it for now because you don't feel it yet, you can add 10 other variables and the lint will never hint again.

Maybe just get rid of the easiest inner function and remove the suppression.

Reply:
29e42118a7cdbfb2487b4aef7f3abcce83b23ed4